features/search/events.feature

```diff
@@ -10,3 +10,15 @@ Feature:
     When I am logged as "benjyd@aol.com"
     And I am on "/evenements"
     Then the "search-city" field should contain "Marseille 3e, France"
+
+  Scenario: The display count participant need logging
+    Given the following fixtures are loaded:
+      | LoadAdherentData |
```

Comment:
You should have the following to load the fixture for all scenarios 

```
Background:
    Given the following fixtures are loaded:
      | LoadAdherentData      |
```

Reply:
Fixed i used the background features